### ISSUE TITLE
diffForHumans in Thai is incorrect.

### ISSUE THREAD

**Issue by nont65** [briannesbitt/Carbon#984]
I try to use diffForHumans  with setLocale('th')

```
Carbon::setLocale('th');

Carbon::now()->diffForHumans(Carbon::createFromFormat('Y-m-d','2017-08-01'), true);
```

Result should be "7 เดือน" but it show "1 เดือน" and I reviewed th.php in lang folder. I try to rewrite it like this code below.

```
return array(
    'year' => ':count ปี',
    'y' => ':count ปี',
    'month' => ':count เดือน',
    'm' => ':count เดือน',
    'week' => ':count สัปดาห์',
    'w' => ':count สัปดาห์',
    'day' => ':count วัน',
    'd' => ':count วัน',
    'hour' => ':count ชั่วโมง',
    'h' => ':count ชั่วโมง',
    'minute' => ':count นาที',
    'min' => ':count นาที',
    'second' => ':count วินาที',
    's' => ':count วินาที',
    'ago' => ':time ที่แล้ว',
    'from_now' => ':time จากนี้',
    'after' => 'หลัง:time',
    'before' => 'ก่อน:time',
);
```

It show correct. Can you updated this code on next updated?

**Comment by andrewfenn** [comment]
It's not clear from your example code what the issue is. Can you provide an example without using Carbon::now()?

**Comment by nont65** [comment]
Thank you for response. This is my example.

`
Carbon::setLocale('th');
echo Carbon::createFromFormat('Y-m-d','2017-08-29')->diffForHumans(Carbon::createFromFormat('Y-m-d','2017-08-01'), true);
`

It should show "4 สัปดาห์" but it show "1 สัปดาห์"

**Comment by kylekatarnls** [comment]
Hi, it's now fixed with 1.23.0.
